### DISCUSSION TITLE
Make Firefly window resize responsively

### DISCUSSION THREAD

**Discussion by antonionardella** [iotaledger/firefly#1561] [Features / Enhancements]
## Description

I am mostly working with the screen divided in half by windows, the default (smallest) Firefly size takes up more than half the screen and it does not autoresize to half the screen if I pull it to the top right/left hot corners that resize almost all other applications.
Trinity could be resized to take up less.

## Motivation

The first thing that comes to my mind, Firefly covers more of half the screen and it would be not as easy to read an address/doublecheck information from an exchange/transaction and one has to _juggle_ between windows

e.g. Checking an address on Bitfinex
![image](https://user-images.githubusercontent.com/7383572/110218783-dad04b00-7ebb-11eb-9580-a7ad11d6734b.png)

![image](https://user-images.githubusercontent.com/7383572/110218807-f0457500-7ebb-11eb-99f4-6786da77a196.png)

This is Firefly's minimal size compared to Trinity

![image](https://user-images.githubusercontent.com/7383572/110218871-5d590a80-7ebc-11eb-98e6-0ae7d22da4b3.png)

## Requirements

Write a list of what you want this feature to do.

1. Make Firefly occupy half the screen or less

## Are you planning to do it yourself in a pull request?

No. I do not have the skills, sorry

**Comment by cvarley100** [comment]
This is planned in a future version.

**Comment by maxwellmattryan** [comment]
Hey @antonionardella - I've converted this into a discussion [here](https://github.com/iotaledger/firefly/discussions/1481). Going to close this issue 👍
